ckb/src/test/java/service/ApiTest.java

```diff
@@ -343,4 +346,39 @@ public void testGetCapacityByLockHash() throws Exception {
     Assertions.assertNotNull(lockHashCapacity.capacity);
     Assertions.assertNotNull(lockHashCapacity.cellsCount);
   }
+
+  @Test
+  public void testBatchRpc() throws IOException {
+    List<RpcResponse> rpcResponses =
+        api.batchRPC(
+            Arrays.asList(
+                Arrays.asList("get_block_hash", "0x200"),
```

Comment:
will the parameter support decimal numbers be better?

Reply:
Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/331/commits/870352625eb661d8f0095eedd1327db0fd5dd1eb